### ISSUE TITLE
PR isn't labeled for Hacktoberfest...does it need to be?

### ISSUE THREAD

**Issue by LarsTheHacker** [bajajvinamr/HacktoberFest2020#616]
Hello! This is my first time participating in Hacktoberfest! I submitted my PR here (which will be my 2nd PR for contributing to Hacktoberfest), and my Digital Ocean profile said that it was ineligible. Is it because I didn't request it with a tag? Or do I need to wait for the author to accept the PR and start the review period?

Any help is appreciated, I just want to know how to play correctly. :)

-Lars

**Comment by thee-moon** [comment]
I am facing similar issues.

**Comment by Mu-C00L** [comment]
I am also facing the same issue, although my PR was merged.

**Comment by RohitKumar-200** [comment]
As described in https://hacktoberfest.digitalocean.com/hacktoberfest-update , it is not necessary to tag individual PR with hacktoberfest-accepted
![Hacktoberfest](https://user-images.githubusercontent.com/55583961/95909664-80273d00-0dbc-11eb-972d-750b3ecf2dd6.png)

This repo has Topic: hacktoberfest 

But, i am also facing similar issue

@bajajvinamr please review it

**Comment by Torfab** [comment]
same for me

**Comment by Anjalikumari15** [comment]
Same issue. For me, it's showing that the repository is ineligible. Is it same for everyone?

**Comment by LarsTheHacker** [comment]
Yes, that's the problem I'm having. Maybe the author has to label the merge once accepted?

**Comment by Goku-kun** [comment]
I think if the owner of this repository accepts every PR and labels them with "hacktoberfest-accepted",  then maybe it would get eligible for hacktoberfest.

**Comment by Vyvy-vi** [comment]
well your PR hasn't been merged

**Comment by Vyvy-vi** [comment]
Issue #633 
Pull requests here are not counted

**Comment by Sunairaa** [comment]
yeah, i also faced the same issue. but what I think label can be added when someone merges it.
